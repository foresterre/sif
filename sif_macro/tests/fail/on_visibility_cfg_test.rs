// see also 'tests/ok/visibility_2'

#[macro_use]
extern crate sif_macro;

#[parameterized]
#[case(0)]
#[case(1)]
fn my_test(a: u8) {}

#[cfg(not(test))]
fn cant_call_since_in_cfg_test_mod() {
    my_test::case_0(); // this is not ok
}

fn main() {}
