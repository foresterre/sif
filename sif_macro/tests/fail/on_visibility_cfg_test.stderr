error[E0433]: failed to resolve: use of undeclared type or module `my_test`
  --> $DIR/on_visibility_cfg_test.rs:13:5
   |
13 |     my_test::case_0(); // this is not ok
   |     ^^^^^^^ use of undeclared type or module `my_test`
